pkg/codegen/internal/test/testdata/simple-resource-schema/nodejs/resource.ts

```diff
@@ -51,6 +51,13 @@ export class Resource extends pulumi.CustomResource {
         if (!opts.version) {
             opts = pulumi.mergeOptions(opts, { version: utilities.getVersion()});
         }
+        // Always mark these fields as secret to avoid leaking sensitive values into the state.
```

Comment:
There is no existing precedent for emitting comments into this codegen.  Suggest we maintain stylistic consistency here and leave off these comments in all languages.

Reply:
Ok, I'll move the comment into the code generators instead.